README.md

```diff
@@ -1,14 +1,97 @@
 # FHIR Patient Generator
 
-A tool for generating eCQM-specific patient data using [Synthea](https://github.com/synthetichealth/synthea)
+A tool for generating eCQM-specific patient data using [Synthea&trade;](https://github.com/synthetichealth/synthea)
 
-# Prerequisites
-* [Node.js](https://nodejs.org/en/)
-* Install the [fhir-bundle-calculator](https://github.com/projecttacoma/fhir-bundle-calculator) command line utility:
+###### Table of Contents
+
+* [Prerequisites](#prerequisites)
+* [Developer Quickstart](#developer-quickstart)
+  * [Installation](#installation)
+  * [Usage](#usage)
+  * [Examples](#examples)
+* [Generation and Calculation](#generation-and-calculation)
+* [Existing Patient Data](#existing-patient-data)
+* [Pre-loading a Docker Image](#pre-loading-a-docker-image)
+
+## Prerequisites
+* [Node.js >=10.15.1](https://nodejs.org/en/)
+* [Docker](https://www.docker.com/)
+* [fhir-bundle-calculator](https://github.com/projecttacoma/fhir-bundle-calculator#installation-with-npm) command line utility
+
+## Developer Quickstart
+
+**NOTE**: This tool is only compatible with `fhir-bundle-calculator` version 3.0.0 or higher:
 
 ``` bash
 npm install -g fhir-bundle-calculator
+calculate-bundles --version
+# should ouput 3.x.x
+```
+
+### Installation
+
+``` bash
+git clone https://github.com/projecttacoma/fhir-patient-generator
+cd fhir-patient-generator
+```
+
+### Usage
+
+``` bash
+make MEASURE_DIR=/path/to/measure/dir <r4|stu3>
+```
```

Comment:
getting weird syntax highlighting on the `3>` since it's trying to interpret that as bash, maybe just omit `bash` in case that could be confusing.